apps/applicant-tracking/src/services/api/applicant/index.ts

```diff
@@ -0,0 +1,18 @@
+import { urls } from '@/utils/constants/urls';
+import { zApplicantResponse } from '@shared/types';
+
+class ApiApplicantService {
+  async get(id: string) {
+    const response = await fetch(urls.api.getApplicant(id));
+    const data = await response.json();
+
+    const validationResult = zApplicantResponse.safeParse(data);
+    if (validationResult.success) {
+      return validationResult.data;
+    }
+
+    return null;
```

Comment:
two cases that could return null here
1. validation failed
2. not found
May want something to distinguish the two?